grow-website-gatsby/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write src/**/*.{js,jsx,ts,tsx}",
+    "format": "prettier --write src/**/*.{js,jsx,ts,tsx} src/**/**/*.{js,jsx,ts,tsx} src/**/**/**/*.{js,jsx,ts,tsx}",
```

Comment:
@catepalmer you should not need to define further levels of nesting, as that's what the ** should handle. I haven't tested this, but as per prettier docs, you should wrap the path in quotes, so that it's prettier processing it rather than the shell :) . Try using `"format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",` instead.